engine/engine/src/engine_service.cpp

```diff
@@ -14,6 +14,7 @@
 
 #include <stdint.h>
 #include <string.h>
+#include <limits>
```

Comment:
Please prefer the C api (i.e. `float.h`)
We try to avoid the C++ standard libraries (templates etc).
It's also about communication, we generally don't want any C++ standard library code in our engine.
You can read about our code style and some of our reasons here:
https://defold.com/2020/05/31/The-Defold-engine-code-style/